examples/basic/catalogValidationValues.json.template

```diff
@@ -0,0 +1,6 @@
+{
+  "ibmcloud_api_key": $VALIDATION_APIKEY,
+  "region": "us-south",
+  "resource_tags": $TAGS,
+  "prefix": $PREFIX
+}
```

Comment:
pass the vars to use existing secrets manager instance. Use `$SM_GUID` and `$SM_REGION` - I'll update pipeline to support this